.github/workflows/wheel_tests_and_release.yml

```diff
@@ -115,9 +119,19 @@ jobs:
       - uses: actions/setup-python@v5
         name: Install Python
         with:
-          python-version: "3.9"
+          python-version: "3.12"
+
+      - name: Build wheels for CPython (macOS) (x86_64)
+        if: matrix.cibw_arch == 'x86_64' && matrix.os == 'macos-latest'
+        uses: pypa/cibuildwheel@8d945475ac4b1aac4ae08b2fd27db9917158b6ce # 2.17.0
+        with:
+          output-dir: dist
+        env:
+          CIBW_BUILD: ${{ matrix.cibw_python }}-*
+          CIBW_ARCHS_MACOS: ${{ matrix.cibw_arch }}
 
-      - name: Build wheels for CPython (macOS)
+      - name: Build wheels for CPython (macOS) (arm64)
+        if: matrix.cibw_arch == 'arm64' && matrix.os == 'macos-14'
```

Comment:
Same here, `&& matrix.os == 'macos-14'` isn't needed.

Reply:
Addressed in 868515d5e0ac1f8038c910415a754ece5189b82a.